dashboard/app/models/levels/level.rb

```diff
@@ -840,6 +840,7 @@ def summarize_for_lab2_properties(script, script_level = nil, current_user = nil
     properties_camelized = properties.camelize_keys
     properties_camelized[:id] = id
     properties_camelized[:levelData] = video if video
+    properties_camelized[:helpVideos] = related_videos.map(&:summarize)
```

Comment:
I wanted to make a distinction between a video for a video level and help & tips videos; this seemed clearer to me. What do other people think?

Reply:
makes sense to me!